scalafix-tests/input/src/main/scala/test/DisableSyntax.scala

```diff
@@ -11,6 +11,7 @@ DisableSyntax.noSemicolons = true
 DisableSyntax.noXml = true
 DisableSyntax.regex = [
   {
+    id = offensive
```

Comment:
I cannot use pimp here because the assert would trigger the regex again.

Reply:
Interesting, I wonder if we can avoid this in general by making sure the offensive part does not follow `scalafix:ok/off`. Since we have the start offset, we should be able to handle this. I opened https://github.com/scalacenter/scalafix/issues/507 to track this.

Reply:
"offensive" is better anyways.